src/gui/Connected.qml

```diff
@@ -119,10 +156,560 @@ Item {
         inviteKeyString: virtualstudio.currentStudio >= 0 ? serverModel[virtualstudio.currentStudio].inviteKey : ""
     }
 
+    Item {
+        id: deviceSettings
+        visible: showReadyScreen && isUsingRtAudio
+        x: bodyMargin * virtualstudio.uiScale; y: 200 * virtualstudio.uiScale
+        width: parent.width - (2 * x)
+        height: 360 * virtualstudio.uiScale
+        clip: true
+
+        Button {
+            id: deviceSettingsButton
+            background: Rectangle {
+                radius: 6 * virtualstudio.uiScale
+                color: deviceSettingsButton.down ? browserButtonPressedColour : (deviceSettingsButton.hovered ? browserButtonHoverColour : browserButtonColour)
+            }
+            onClicked: popup.open()
+            anchors.right: parent.right
+            width: 144 * virtualstudio.uiScale; height: 24 * virtualstudio.uiScale
+
+            Text {
+                text: "Change Device Settings"
+                font { family: "Poppins"; pixelSize: fontTiny * virtualstudio.fontScale * virtualstudio.uiScale}
+                anchors { horizontalCenter: parent.horizontalCenter; verticalCenter: parent.verticalCenter }
+                color: textColour
+            }
+        }
+
+        Popup {
+            id: popup
+            padding: 1
+            width: parent.width
+            height: parent.height
+            anchors.centerIn: parent
+            modal: true
+            focus: true
+            closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutsideParent
+            onClosed: {
+                virtualstudio.applySettings()
+            }
+
+            background: Rectangle {
+                anchors.fill: parent
+                color: "transparent"
+                radius: 6 * virtualstudio.uiScale
+                border.width: 1
+                border.color: buttonStroke
+                clip: true
+            }
+
+            contentItem: Rectangle {
```

Comment:
I initially had it in here, but a few things didn't work so well because:
* AudioSettings is built for a full-screen experience and didn't fit in the popup
* There's a few things within AudioSettings that imo don't make sense within the connected view. Specifically things like volume control (which is already on the connected view) and the "play test tone" button (which again, you're already connected)